react/components/ExtensionPointComponent.tsx

```diff
@@ -158,15 +159,15 @@ class ExtensionPointComponent extends PureComponent<
 
     // A children of this extension point throwed an uncaught error
     // Only show errors in production if the entire page explodes. (Ignore nested extension points)
-    if (error) {
+    if (error && errorInfo) {
```

Comment:
`ExtensionPointError` accepts `errorInfo` as undefined.

```tsx
public render() {
    const {treePath, error, errorInfo} = this.props
    const {errorDetails} = this.state
    const componentStack = errorInfo && errorInfo.componentStack
```

Changing this `if` can change the current behavior.